technic/machines/HV/nuclear_reactor.lua

```diff
@@ -209,9 +226,21 @@ local function reactor_structure_badness(pos)
 	return (25 - water_layer) + (96 - lead_layer) + (216 - blast_layer)
 end
 
+local mcl_expl_info = {
+   drop_chance = 1.0,
+   max_blast_resistance = 10,
+   sound = true,
+   particles = true,
+   fire = true,
+   griefing = true,
+   grief_protected = true,
+}
 
 local function melt_down_reactor(pos)
-	minetest.log("action", "A reactor melted down at "..minetest.pos_to_string(pos))
+        minetest.log("action", "A reactor melted down at "..minetest.pos_to_string(pos))
+	if minetest.get_modpath("mcl_explosions") then
+	  mcl_explosions.explode(pos, 30, mcl_expl_info)
+	end
```

Comment:
Just indentation / style
```suggestion
	if minetest.get_modpath("mcl_explosions") then
		mcl_explosions.explode(pos, 30, mcl_expl_info)
	end
```